improver/cube_combiner.py

```diff
@@ -140,6 +167,57 @@ def _get_expanded_coord_names(cube_list):
                     expanded_coords.append(coord)
         return expanded_coords
 
+    def _setup_coord_broadcast(self, cube_list):
```

Comment:
Maybe `_broadcast_to_coord` - says what it's doing a bit better?

This is also part of the slightly wider inconsistency - is `broadcast_coord` a single string value (or `None`) or a list that might contain multiple values?

Reply:
I've gone with "_setup_coords_for_broadcast" as this doesn't do the actual broadcasting, that's done deep down in Numpy somewhere. This method just makes it possible.

Reply:
OK